src/app/shared/user-navbar/users.config.ts

```diff
@@ -0,0 +1,5 @@
+const users = {
+  admin: ['Wydarzenia', 'Propozycje kategorii', 'Kategorie', 'Grupy', 'Hashtsagi', 'Wyloguj'],
+  user: ['Wydarzenia', 'Moje wydarzenia', 'Grupy', 'Moje grupy', 'Wyloguj'],
+};
+export default users;
```

Comment:
Trzeba by też dodać gdzieś miejsce na linki do poszczególnych route'ów. Nie jestem pewny, ale może warto by było lepiej to otypować?